addons/project_mrp_account/data/project_mrp_account_demo.xml

```diff
@@ -18,24 +18,22 @@
             <field name="location_src_id" ref="stock.stock_location_stock"/>
             <field name="location_dest_id" ref="stock.stock_location_stock"/>
             <field name="bom_id" ref="mrp.mrp_bom_drawer"/>
-            <field name="analytic_distribution" eval="{ref('project.analytic_construction'): 100}"/>
         </record>
         <record id="mrp_production_dinning_table_2" model="mrp.production">
-            <field name="product_id" ref="project_mrp.product_product_dinning_table"/>
+            <field name="product_id" ref="project_mrp_account.product_product_dinning_table"/>
```

Comment:
Why do we need to move those data in `project_mrp_account`? 🧐

Reply:
The previous project_mrp module was depending on ['mrp_account', 'project'], the new one only depends on ['mrp', 'project'].
We then created a new bridge project_mrp_account depending on ['mrp_account', 'project_mrp'] (like the previous project_mrp module), so we had to move the data to this new bridge module.